app/src/main/java/net/squanchy/SquanchyApplication.java

```diff
@@ -30,6 +30,7 @@ public void onCreate() {
         setupTracking();
         FirebaseDatabase.getInstance().setPersistenceEnabled(true);
         TypefaceManager.init();
+        applicationComponent().service();
```

Comment:
There was an issue with the fact that dagger instantiated the nearITmanager too late (lazy loading), missing an application lifecycle event that prevented the beacon scans to work on the first app run, and every time after an app swipe from the task manager.